source/MaterialXTest/MaterialXRuntime/Runtime.cpp

```diff
@@ -784,9 +784,10 @@ TEST_CASE("Runtime: NodeGraphs", "[runtime]")
     const mx::RtToken ADDGRAPH("addgraph");
     const mx::RtToken MATH_GROUP("math");
     const mx::RtToken ADDGRAPH_VERSION("3.4");
+    const mx::RtToken ADDGRAPH_VERSION2("3.5");
```

Comment:
Is this used anywhere? Don't seem to see it.

Reply:
Seems not. I'll remove.